### DISCUSSION TITLE
Add support for sentry

### DISCUSSION THREAD

**Discussion by mehrdad-khojastefar** [jertel/elastalert2#1307] [Ideas]
I was wondering if there are any plans for integrating sentry into elastalert or not?
The problem I was facing was that elastAlert ( deployed on k8s ) terminated and I did not have access to the logs. 
I thought that it would be very good to have sentry enabled so that I knew what went wrong. 
I would be happy to integrate this feature to the elastalert.

**Comment by mehrdad-khojastefar** [comment]
And since we are relying on sentry for error monitoring and other stuff that is happening in our infrastructure, I think it would be very helpful to have elastalert in our sentry as well.

**Reply by jertel** [reply]
There is no plan at this time to integrate Sentry. However, I'm open to you adding this depending on the complexity and approach.

Could you put together a brief, high level outline of what you intend to do to support this? E.g., which Python code will be modified, any Helm chart changes, how you plan to test, etc?

**Reply by nsano-rururu** [reply]
https://github.com/Yelp/elastalert/pull/1716

**Comment by mehrdad-khojastefar** [comment]
Hi @jertel any plan on integrating this feature?

**Reply by jertel** [reply]
Hello @mehrdad-khojastefar,

If you're up for submitting a PR, following the [contribution guidelines](https://github.com/jertel/elastalert2/blob/master/CONTRIBUTING.md), I would be willing to help review it and give pointers where changes are needed.

**Reply by mehrdad-khojastefar** [reply]
I will try to free up some time on weekend and add this feature then.
More than happy to help.
